types/src/system_txn/mod.rs

```diff
@@ -15,3 +15,17 @@ pub enum SystemTransaction {
 pub struct DummySystemTransaction {
     pub nonce: u64,
 }
+
+impl SystemTransaction {
+    pub fn dummy(nonce: u64) -> Self {
```

Comment:
isn't Arbitrary a testing feature? something like this doesn't work? #[cfg(any(test, feature = "fuzzing"))]

Reply:
fixed